desktop/apps/auction/components/basic_checkbox/index.js

```diff
@@ -1,13 +1,13 @@
-import classNames from 'classnames'
+import PropTypes from 'prop-types'
 import React from 'react'
+import classNames from 'classnames'
```

Comment:
Like here, `c` comes before `P`? (I promise I was attempting to keep them organized haha).